subgraphs/uniswap-v3-forks/protocols/sushiswap-v3/config/deployments/sushiswap-v3-moonriver/configurations.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Address,
+  BigDecimal,
+  BigInt,
+  Bytes,
+  log,
+} from "@graphprotocol/graph-ts";
+import { Factory } from "../../../../../generated/Factory/Factory";
+import {
+  BIGDECIMAL_ONE,
+  BIGDECIMAL_ZERO,
+  FeeSwitch,
+  Network,
+  RewardIntervalType,
+} from "../../../../../src/common/constants";
+import { Configurations } from "../../../../../configurations/configurations/interface";
+import { PROTOCOL_NAME, PROTOCOL_SLUG } from "../../../src/common/constants";
+import { stringToBytesList } from "../../../../../src/common/utils/utils";
+
+export class SushiswapV3MoonriverConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.MOONRIVER;
+  }
+  getProtocolName(): string {
+    return PROTOCOL_NAME;
+  }
+  getProtocolSlug(): string {
+    return PROTOCOL_SLUG;
+  }
+  getFactoryAddress(): Bytes {
+    return Bytes.fromHexString("0x2f255d3f3c0a3726c6c99e74566c4b18e36e3ce6");
+  }
+  getFactoryContract(): Factory {
+    return Factory.bind(
+      Address.fromString("0x2f255d3f3c0a3726c6c99e74566c4b18e36e3ce6")
+    );
+  }
+  getProtocolFeeOnOff(): string {
```

Comment:
Is the protocol fee currently on? You might need to flip these on if so.

Reply:
https://github.com/messari/subgraphs/pull/2151/commits/6e49600a46e33ab691a00d2a15f58f8053113ce3